apps/src/storage/dataBrowser/PreviewModal.jsx

```diff
@@ -40,7 +40,11 @@ class PreviewModal extends React.Component {
         <div style={{overflow: 'scroll', maxHeight: '70%'}}>
           <DataTable readOnly rowsPerPage={100} />
         </div>
-        <button type="button" onClick={() => this.importTable(datasetInfo)}>
+        <button
+          id="ui-test-import-table-btn"
+          type="button"
+          onClick={() => this.importTable(datasetInfo)}
+        >
```

Comment:
I added a ui test id here because [its associated test](https://github.com/code-dot-org/code-dot-org/pull/55365/files#diff-2e21c2dfc0a6a6df6c47c18ed323891bed9832aeb87a3b2ebc30257eed69dcfbR19) wasn't working by simply incrementing the index of which button on the modal it shoudl click.